db/evidences.go

```diff
@@ -0,0 +1,32 @@
+package db
+
+import (
+	"github.com/flanksource/incident-commander/api"
+	"github.com/google/uuid"
+
+	"github.com/flanksource/commons/logger"
+)
+
+func GetEvidenceScripts() []api.Evidence {
+	var evidences []api.Evidence
+	openIncidentsSubQuery := Gorm.Table("hypotheses").Select("id").Where("incident_id IN (?)",
+		Gorm.Table("incidents").Select("id").Where("closed IS NULL AND resolved IS NULL"))
+	err := Gorm.Table("evidences").
+		Joins("Config").Joins("Component").
+		Where("evidences.hypothesis_id IN (?)", openIncidentsSubQuery).
+		Where("evidences.script IS NOT NULL").
+		Where("definition_of_done != true").
```

Comment:
I think this should be other way around - only definition of done evidence items can have scripts attached